README.md

```diff
@@ -1,2 +1,32 @@
 # mongo-collector-metrics
 this repository was created for the mongodb metrics collector, this app sends metrics to the prometheus
+
+## Mongodb Exporter
+
+Based on MongoDB exporter for prometheus.io, written in go (https://github.com/dcu/mongodb_exporter), but forked for full sharded support and structure changes.
+
+## Features
+
+- MongoDB Server Status metrics (cursors, operations, indexes, storage, etc)
+- MongoDB Replica Set metrics (members, ping, replication lag, etc)
+- MongoDB Replication Oplog metrics (size, length in time, etc)
+- MongoDB Sharding metrics (shards, chunks, db/collections, balancer operations)
+- MongoDB RocksDB storage-engine metrics (levels, compactions, cache usage, i/o rates, etc)
+- MongoDB WiredTiger storage-engine metrics (cache, blockmanger, tickets, etc)
+
+## Install 
+
+We run Deploy in production of the mongo-collector-metrics application, as follows:
+
+`kubectl apply -f https://github.com/GrupoZapVivaReal/vivareal-graylog/tree/master/deploy-k8s/40-secrets/secret-mongo-collector-metrics.yaml`
+
+`kubectl apply -f https://github.com/GrupoZapVivaReal/vivareal-graylog/tree/master/deploy-k8s/90-collector-mongo `
+
+After that, we access the link http://prometheus.private.prod.vivareal.io and run the following query `mongodb_mongod_connections`, then see the result below as evidence that the logs are being sent to prometheus:
```

Comment:
The `PROD` URL shouldn't be fixed, gives the impression that `QA` doesn't exist.
Maybe use a placeholder like `http://prometheus.private.<env>.vivareal.io`